docs/views/spec/spec.mdx

```diff
@@ -39,11 +44,13 @@ The Specification is defined as follows:
-| label       | Optional[[ViewElement](/docs/views/spec/other)] | Specifying how to interpret and display the label column of your Zeno Project.                                                                                                                                                                                                                     |
-| output      | Optional[[ViewElement](/docs/views/spec/other)] | Specifying how to interpret and display the ouptut column of your Zeno Project.                                                                                                                                                                                                                    |
-| displayType | Optional[String]                                | The type of display this view uses. If this parameter is omitted, the default view with three sections for data, label, and output will be used. If set as `"table"`, data, label, and output must be a [TableElement](/docs/views/spec/table) instead of a [ViewElement](/docs/views/spec/other). |
+| Property    | Type                                                                            | Description                                                                                                                                                                              |
+| ----------- | ------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| data        | [ViewElement](/docs/views/spec/other) \| [TableElement](/docs/views/spec/other) | How to render data entries.                                                                                                                                                              |
+| label       | [ViewElement](/docs/views/spec/other) \| TableElement                           | How to render label entries.                                                                                                                                                             |
+| output      | Optional[[ViewElement](/docs/views/spec/other)]                                 | How to render output entries.                                                                                                                                                            |
+| displayType | Optional[String]                                                                | How to display data, label, and output entries. The default view is a vertical list. If set to `"table"`, data, label, and output must a [TableElement](/docs/views/spec/table) entries. |
```

Comment:
Maybe we can just be clear here that if default, only use ViewElement, if table, only use TableElement.